app/assets/javascripts/admin/addon/components/admin-flags.gjs

```diff
@@ -0,0 +1,27 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { inject as service } from "@ember/service";
+import i18n from "discourse-common/helpers/i18n";
+import AdminFlag from "admin/components/admin-flag";
+
+export default class AdminFlags extends Component {
+  @service site;
+  @tracked flags = this.site.flagTypes;
```

Comment:
I don't think this needs to be tracked since the reactivity is limited to the sub-component, try doing `flags = this.site.flagTypes` here